### PR TITLE
Patch to set the content_type implicitly or explicitly while uploadin…

### PR DESCRIPTION
**Pull request by dipatidar**
path in reference to #28.  Patch to set the content_type implicitly or explicitly while uploading content.

### CODE REVIEW

**Comment by darenr** [comment]
once you've added the tests can you post the screenshots showing both implicit and explicit results for some file types (one good one to use is README.md since mimetypes doesn't have a value for this (resulting in your [0] picking None)

**Comment by darenr** [comment]
this looks good, I did notice the precommit hooks were not running - if you run `pre-commit install` and commit the result then this looks good unless @ahosler sees anything that he has a concern about.

File: ocifs/tests/test_spec.py [hunk 1]
```diff
@@ -1507,3 +1508,69 @@ def test_sync(fs):
...
+
+    import io
+
+    e_path = "image.jpeg"
+    e = os.path.join(full_test_bucket_name, e_path)
+
+    image = Image.new("RGBA", size=(50, 50), color=(256, 0, 0))
+    image_file = io.BytesIO(image.tobytes())
+
+    if isImplicit:
```
Comment (mayoor) [inline ocifs/tests/test_spec.py:1527]: If/else inside the test case is confusing. Can you break this into two test cases.

**Review by ahosler — APPROVED** [review]
Looks great, thanks!

**Comment by ahosler** [comment]
image
Unit tests passed 👍

**Pull request merged**